src/components/feature/auth/login/LoginForm.jsx

```diff
@@ -14,6 +14,8 @@ import * as C from "@utils/constants";
 
 import * as S from "./styles";
 import { Loader } from "@components/shared/custom-loadings/Loader";
+import { CustomInput } from "@components/shared/custom-inputs";
+import { formatDate } from "@components/shared/custom-table/utils";
```

Comment:
akurat to jest bezsensu :D z powodu takiego, że utils dla custom-table jest i wyłącznie dla niego.